core/src/mindustry/input/InputHandler.java

```diff
@@ -544,7 +544,7 @@ public void drawBreaking(int x, int y){
     }
 
     public void useSchematic(Schematic schem){
-        selectRequests.addAll(schematics.toRequests(schem, player.tileX(), player.tileY()));
+        selectRequests.addAll(schematics.toRequests(schem, (int)Core.camera.position.x, (int)Core.camera.position.y));
```

Comment:
This should be in MobileInput, not input handler. Instead of casting to an int, use `World.toTile` to properly convert from world to tile coordinates